config/initializers/99_clowder-config.rb

```diff
@@ -0,0 +1,37 @@
+
+if ClowderCommonRuby::Config.clowder_enabled?
+
+  config = ClowderCommonRuby::Config.load
+
+  Settings.add_source!({kafka_producer_topics.inventory_events: config.kafka_topics['platform.inventory.events']})
```

Comment:
This is not a valid symbol. I believe you need to nest the hashes:

```suggestion
  Settings.add_source!({kafka_producer_topics: { inventory_events: config.kafka_topics['platform.inventory.events']}})
```

Really, it'd be better to add only one source for clowder:

```
Settings.add_source!(Rails.root.join('config', 'clowder.yml'))
```

Then, you can use erb within clowder.yml to set any values you'd like.

Reply:
Or, (if the YAML wouldn't look okay) you can try to construct a single Hash.

Reply:
Yes, to be clear, Victor, one single gigantic hash for all the clowder config; Only one `add_source!` call.

Reply:
gotcha, adapted to the "big hash" approach